Automattic/sensei
# Switch icon to SVG for Calypso compatibility

Currently we don't set an icon in add_menu_page but instead use jQuery to display the icon.  This chnage will make our icon compatible with Calypso so that it will also display the proper icon.

### Changes proposed in this Pull Request

* Switches our icon from a font set with jQuery to just using an SVG icon with `add_menu_page`.

### Testing instructions

* Use this patch
* Go to your dashboard
* Make sure the icon is still there

**To test on Calypso:** 

* Go to a WordPress.com Atomic site
* Install this patched version OR install Sensei LMS and patch the file with these changes
* Make sure that the icon looks good on this url: https://wordpress.com/hosting-config/{YOUR ATOMIC SITE URL}

### Screenshot / Video

**Previously:**
<img width="335" alt="Screenshot 2022-11-22 at 3 47 13 PM" src="https://user-images.githubusercontent.com/3220162/203443691-37da5fbf-8509-4576-81cc-c8774803c18a.png">

**Now:** 
<img width="283" alt="Screenshot 2022-11-22 at 3 47 26 PM" src="https://user-images.githubusercontent.com/3220162/203443702-3cf2ddba-d277-4ec1-9e28-8577e1d7aff7.png">
